diff --git a/usr/share/openmediavault/scripts/proftpd b/usr/share/openmediavault/scripts/proftpd
index ff94bc0..0956af8 100755
--- a/usr/share/openmediavault/scripts/proftpd
+++ b/usr/share/openmediavault/scripts/proftpd
@@ -31,8 +31,8 @@ set -e
 
 OMV_PROFTPD_CONFIG=${OMV_PROFTPD_CONFIG:-"/etc/proftpd/proftpd.conf"}
 OMV_PROFTPD_EXTENSIONSDIR=${OMV_PROFTPD_EXTENSIONSDIR:-"${OMV_SCRIPTS_DIR}/proftpd.d"}
-OMV_PROFTPD_DEFAULTROOT=${OMV_PROFTPD_DEFAULTROOT:-"/home/ftp"}
-OMV_PROFTPD_DISPLAYLOGIN=${OMV_PROFTPD_DISPLAYLOGIN:-"/home/ftp/welcome.msg"}
+OMV_PROFTPD_DEFAULTROOT=${OMV_PROFTPD_DEFAULTROOT:-"/srv/ftp"}
+OMV_PROFTPD_DISPLAYLOGIN=${OMV_PROFTPD_DISPLAYLOGIN:-"/srv/ftp/welcome.msg"}
 OMV_PROFTPD_USEIPV6=${OMV_PROFTPD_USEIPV6:-"on"}
 OMV_PROFTPD_DEFERWELCOME=${OMV_PROFTPD_DEFERWELCOME:-"off"}
 OMV_PROFTPD_MULTILINERFC2228=${OMV_PROFTPD_MULTILINERFC2228:-"on"}
